src/commands/plugins.rs

```diff
@@ -335,7 +437,7 @@ pub struct List {
 impl List {
     pub async fn run(self) -> Result<()> {
         let mut plugins = if self.installed {
-            Self::list_installed_plugins()
+            list_installed_plugins()
         } else {
             Self::list_catalogue_and_installed_plugins().await
```

Comment:
Although not required for this PR, I would move `list_catalogue_and_installed_plugins` to be a module function too, so that a reader coming to this in 6 months is not puzzled by "why is one of these functions module level and the other one associated?"

Reply:
Hmm, that makes sense!